### ISSUE TITLE
STAR

### ISSUE THREAD

**Issue by luckytanyy** [VisionXLab/STAR-MMRotate#6]
The pixel sizes of the data in the STAR dataset vary. Do we need to preprocess the data? If data preprocessing is required, can you provide the relevant code for data preprocessing.

**Comment by yangxue0827** [comment]
Basically, the data preprocessing is the same as [DOTA](https://github.com/yangxue0827/STAR-MMRotate/tree/main/tools/data/dota).
Note that for STAR, the validation set does not need to be processed because we provide online large-scale image evaluation.

**Comment by mason1008** [comment]
@yangxue0827 “Note that for STAR, the validation set does not need to be processed because we provide online large-scale image evaluation.” 

1. Training is consistent with DOTA, but what are the specific differences between validation and DOTA? How should this be implemented in detail? 
2. When will the code for HOD-Net be released as open source? 

Thank you

**Comment by yangxue0827** [comment]
1. Refer to test set  in config.
https://github.com/yangxue0827/STAR-MMRotate/blob/08a475bd0f6cf2ce919c99cc75093fafa5278d6a/configs/h2rbox_v2p/h2rbox_v2p_r50_fpn_1x_star_le90.py#L102-L109
2. It will be released after the paper accepted.

**Comment by mason1008** [comment]
1. I used tools/data/dota/split/img_split.py to process the STAR dataset during both training and testing. When training with oriented_rcnn_r50_fpn_1x_star_le90.py, there is no difference in the results between test=dict(type='STARWSOODDataset', ...) and test=dict(type='STARDataset', ...) when train=dict(type='STARDataset', ...) is used.
2. In Table 2 of the paper, the mAP of HOD-Net is 43.6 and 55.9, corresponding to single-scale or multi-scale for DOTA ?
3. In Table 3 of the paper, is the only difference between Resizing and Cropping the use of model = dict(type='OrientedRCNN', ...) and model = dict(type='OrientedRCNNCrop', ...) ?

**Comment by yangxue0827** [comment]
1. `STARWSOODDataset` is for weakly supervised detectors (such as H2RBox-v2), and `STARDataset` is for fully supervised detectors.

2. Multi-scale, involving data multi-scale cropping

3. `OrientedRCNNCrop` is specifically for STAR data. The difference from `OrientedRCNN` is that the former can support online evaluation of large images, so there is no need to crop the validation/test set.

**Comment by luckytanyy** [comment]
I would like to ask a question about the test set. If the test set is cropped 1024x1024 and uploaded to the evaluation website, and the result is empty, can the test set be cropped?
